bench/workgen/runner/example_dynamic_tables.py

```diff
@@ -27,139 +27,54 @@
 session = connection.open_session()
 
-# List of all tables.
-tables = []
-
-# Create a table.
-table_config = 'key_format=S,value_format=S'
-table_name = 'table:simple'
-session.create(table_name, table_config)
-tables.append(table_name)
-
 key = Key(Key.KEYGEN_APPEND, 10)
 value = Value(40)
 
-# Create an operation dedicated to one table.
-op = Operation(Operation.OP_INSERT, Table(table_name), key, value)
-thread = Thread(op)
-
 # Create operations that work on random tables.
 op_ins_rnd = Operation(Operation.OP_INSERT, key, value)
 op_upd_rnd = Operation(Operation.OP_UPDATE, key, value)
 op_read_rnd = Operation(Operation.OP_SEARCH, key, value)
 thread_ins_rnd = Thread(op_ins_rnd * 10)
-thread_upd_rnd = Thread(op_upd_rnd * 10)
-thread_read_rnd = Thread(op_read_rnd * 10)
-
-workload = Workload(context, thread + thread_ins_rnd + thread_upd_rnd + thread_read_rnd)
-workload.options.run_time = 10
-
-# Start the workload.
-workload_thread = ThreadWithReturnValue(target=workload.run, args=([connection]))
-workload_thread.start()
-
-# Create and drop tables while the workload is running.
-tables_to_delete = []
-while workload_thread.is_alive():
-    create(session, workload, table_config)
-
-    # Select a random table to drop.
-    if len(tables):
-        idx = random.randint(0, len(tables) - 1)
-        uri = tables[idx]
-        # Mark it for deletion.
-        drop(workload, uri)
-
-    # Trigger the Workgen garbage collection.
-    tables_to_delete += workload.garbage_collection()
-    deleted_tables = []
-    for t in tables_to_delete:
-        try:
-            session.drop(t)
-            # If the WiredTiger call is successful, this table has been removed safely from all
-            # layers.
-            deleted_tables.append(t)
-        except WiredTigerError as e:
-            assert "device or resource busy" in str(e).lower()
-
-    # Update the list of tables pending deletion.
-    for t in deleted_tables:
-        tables_to_delete.remove(t)
-
-    time.sleep(1)
-
-assert workload_thread.join() == 0
-
-# It is possible that Python and Workgen are not matching at this point in terms of tables. Since
-# Workgen cannot remove a table that is still being used, it may have kept reference to it.
-# Now the workload is finished, call the garbage collector on Workgen, nothing should be blocking
-# the deletion of tables that are supposed to be removed.
-# Note that we are not checking what's on disk, we don't need to use what garbage_collection
-# returns.
-workload.garbage_collection()
-
-# Check tables match between Python and Workgen.
-workgen_tables = workload.get_tables()
-
-assert len(tables) == len(workgen_tables)
-for t in tables:
-    assert t in workgen_tables
+thread_upd_rnd = Thread(op_upd_rnd * 5)
+thread_read_rnd = Thread(op_read_rnd * 5)
+
+workload = Workload(context, thread_ins_rnd + thread_upd_rnd + thread_read_rnd)
+workload.options.run_time = 300
+
+# Start a workload thread to create tables periodically:
```

Comment:
I made a few changes to the wording & formatting to hopefully add clarity. What do you think?
```
# Define a workload thread to create tables periodically:
#   - Start creating tables when the database size falls below 100 MB.
#   - Create 3 tables every 5 seconds until the database size reaches 200MB.
#   - Stop creating tables when the database size exceeds 200 MB.
```

Reply:
Done.